Dockerfile.worker

```diff
@@ -1,51 +0,0 @@
-FROM python:3.9 AS downloader
```

Comment:
does moving this into the other dockerfile change any instructions in the `mavedb-deployment` repo?

Reply:
Yeah it will, I'll open a PR over there. It will probably be useful to try that out on staging prior to merging to work out any kinks.

Reply:
I actually just looked into it and now I don't think these changes will actually necessitate any changes over in the deployment repo, which is awesome. We can confirm once we deploy this to staging.